index.bs

```diff
@@ -2669,10 +2686,59 @@ The <dfn export for=commands>browser.createUserContext</dfn> command creates a
 
 <div algorithm="remote end steps for browser.createUserContext">
 
-The [=remote end steps=] are:
+<div algorithm>
+To <dfn>restore insecure certificates behavior</dfn> given |user context|:
+
+1. If [=endpoint node=]'s [=accept insecure TLS=] flag is true,
+   [=override insecure certificates behavior=] with |user context| and true.
+
+1. Otherwise, [=override insecure certificates behavior=] with |user context| and
+   false.
+
+</div>
+
+<div algorithm>
+To <dfn>override insecure certificates behavior</dfn> given |user context| and
+|acceptInsecureCerts|:
+
+1. If |acceptInsecureCerts| is true:
+
+   1. If [=endpoint node=] doesn't support accepting insecure TLS connections,
+      return [=error=] with [=error code=] [=unsupported operation=].
+
+   1. Perform implementation-defined steps to disable all the TLS certificates checks
```

Comment:
nit: I am not sure we can and would be disabling all possible checks (i.e., we still fail if certificate is malformed). I think what we want is to allow self-signed and expired certificates so perhaps we could be more precise here.

Reply:
Good point. I added the specific step of certificate validation to skip.